serverless.yml

```diff
@@ -160,6 +206,7 @@ custom:
   envEncryptionKeyId: # KMS Key used for encrypting values
     local: ${file(kms_key.yml):key} # Key used for local-stage
     dev: ${file(kms_key.yml):key} # Key used for development-stage
+    test: ${file(kms_key.yml):key} # Key used for test-stage
```

Comment:
I suggest to store keys in `kms_key.yml` for each stage:

```yml
dev: <key>
test: <key>
...
```
and use this to get key: `dev:${file(kms_key.yml):dev}`

So, no need to replace the `key` field value with new key if you want to deploy to another stage.

Reply:
Agreed and done 👍